LayoutTests/fast/css/no-style-change-no-repaint.html

```diff
@@ -0,0 +1,19 @@
+<style>
+html {
+    color: black;
+}
+</style>
+<div></div>
+<script>
+window.onload = function() {
+    if (window.internals)
+        internals.startTrackingRepaints();
+    
+    document.body.offsetWidth;
+    document.styleSheets[0].cssRules[0].style.color = 'black';
+    
+    if (window.internals)
+        document.getElementById('layers').textContent = internals.layerTreeAsText(document, internals.LAYER_TREE_INCLUDES_REPAINT_RECTS);
+}
+</script>
+<pre id="layers"></pre>
```

Comment:
Can you transform this test in a reftest? The -ref will be pretty similar with the test itself, but without the <script> part and with hard-coded `textContent` written into the `id="layers"` element. In that way, the -expected file don't need to be a render dump.
reference: https://web-platform-tests.org/writing-tests/reftests.html

Reply:
Done.